consumers/sender/main.go

```diff
@@ -32,8 +33,8 @@ var (
 
 // Cache prefix and duration parameters
 const (
-	CachePrefix   = "sender:"
-	CacheDuration = 300000 * time.Millisecond
+	cachePrefix   = "sender:"
+	cacheDuration = 300000 * time.Millisecond
```

Comment:
Same here, leave a comment indicating how much time is this. Also we should set it to 7 days.